### DISCUSSION TITLE
Some experience share on the installation process for the contributors.

### DISCUSSION THREAD

**Discussion by ahmetakman** [lava-nc/lava#200] [General]
Hello everyone, 

I have been trying to install the framework for 2 days extensively to my two different computers and I would like to share some point for further adjustments on the framework and documentation. 
General;

It seems the installation instructions in the lava-nc.org and the github repository readme seems different.  ( I do no know which one is the updated one).
[Suggestion]
** Just like the pytorch and the nengo websites, if lava become a simple PyPI packet it would be easier for everyone to install.

For Windows 10 9th Gen i7 Python 3.9.6 System:

First of all , currently I am not able to install lava on my Windows computer. I have got several problems and it still continues. 

( this might be stemmed from my lack of experience ) There is no builtin `source` command in Windows PowerShell. So one need to write `python3_venv\Scripts\activate`  to the command line to activate virtual environment.

for the command `pip install e .`  build fails in the step of preparing metadata. ( it was expressed in the previous discussions ) 

For Ubuntu 20.04 1st Gen i7 Python 3.8 System:
(Probably) I have achieved to install but there is a problem of running unittests by `pyb -E unit` . There occurs bunch of warnings and it stops doing anything (for a long time). 

Then I have run the unittests by `python -m unittest` and it run 205 unittests ( I do not know why but skipping 6 of them )

Another important thing is it seems scipy is a requirement but not included in the requirement.txt

EDIT: I noticed that I did not achieve to install it on Ubuntu as well. So, I have installed from binary for the both OSs. Now it seems, Ubuntu is OK. But, trying to solve the problems raised in the Windows (if it does stem from the framework i will update here). 

Thanks,

**Comment by mgkwill** [comment]
Hi @ahmetakman 

Thanks for noticing some issues. I've opened a PR to fix some of the doc issues:
https://github.com/lava-nc/lava/pull/202

**Reply by mgkwill** [reply]
Feel free to open bug issues and push a PR if you find something is broken.

**Reply by ahmetakman** [reply]
Thanks, mate. I am exploring the repo right now so I will when I get used to it.

**Comment by mgkwill** [comment]
"Another important thing is it seems scipy is a requirement but not included in the requirement.txt"

@ahmetakman where is scipy not a requirement? The change that introduced scipy as a requirement added it as a requirement: https://github.com/lava-nc/lava/blob/main/requirements.txt#L6

**Reply by ahmetakman** [reply]
Oh that is my bad, in the ubuntu installation some unit test scripts needed scipy and it was not installed so I assumed it was not in the list. Probably I removed that line mistakenly and it was not installed.
